frontend/awx/views/jobs/hooks/useJobHeaderActions.tsx

```diff
@@ -0,0 +1,51 @@
+import { ButtonVariant } from '@patternfly/react-core';
+import { MinusCircleIcon, RocketIcon } from '@patternfly/react-icons';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { IPageAction, PageActionSelection, PageActionType } from '../../../../../framework';
+import { UnifiedJob } from '../../../interfaces/UnifiedJob';
+import { isJobRunning } from '../jobUtils';
+import { useCancelJobs } from './useCancelJobs';
+import { useRelaunchJob } from './useRelaunchJob';
+
+export function useJobHeaderActions(onComplete: (jobs: UnifiedJob[]) => void) {
+  const { t } = useTranslation();
+  const cancelJobs = useCancelJobs(onComplete);
+  const relaunchJob = useRelaunchJob();
+
+  return useMemo<IPageAction<UnifiedJob>[]>(() => {
+    const cannotCancelJob = (job: UnifiedJob) => {
```

Comment:
This method is already defined in https://github.com/ansible/ansible-ui/blob/main/frontend/awx/views/jobs/hooks/useJobRowActions.tsx#L29-L35, would it be a good idea to move it to some utils dir?

Reply:
@jerabekjiri  I went ahead and updated the code to have the function be exported in the header actions and imported into the row actions.